packages/blade/src/components/Box/styledProps/getStyledProps.ts

```diff
@@ -54,7 +54,9 @@ const makeStyledProps = (props: StyledPropsInputType): KeysRequired<StyledPropsB
     display: props.display,
     justifySelf: props.justifySelf,
     placeSelf: props.placeSelf,
+    placeItems: props.placeItems,
```

Comment:
We can skip these props from styled-props. They are only relevant for Box no

Reply:
Since we already allow `placeSelf`, makes sense to add `placeItems` as well. Can skip `pointerEvents` though. Not sure.

Reply:
No for example `placeItems` is for the scenario where we place the items inside of the container in center. We don't want people to move the items inside of the button.

Which is why we support `alignSelf` and `justifySelf` on styled props but we don't support `alignItems` and justifyItems` on it